types/CommonTypes.ts

```diff
@@ -14,7 +14,7 @@ import { CharacterStyle } from './CharacterStyleTypes';
 export type ConfigType = {
     onStateChanged?: (state: InitialStateType) => void;
     onSelectedFrameLayoutChanged?: (state: FrameLayoutType) => void;
-    onSelectedFrameContentChanged?: (state: FrameType) => void;
+    onSelectedFrameContentChanged?: (state: Frame) => void;
```

Comment:
This `FrameType` is pretty confusing, and I think obsolete? Can you create a follow up to double check that?